keyboards/otaku_split/rev1/otaku_split.h

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2019 takashiski
```

Comment:
This file should be named `rev1.h`, and that's why it wasn't working with `QMK_KEYBOARD_H`.